samples/bluetooth/hap_ha/Kconfig

```diff
@@ -19,4 +19,34 @@ config HAP_HA_SET_RANK
 	help
 	  Rank of this device in set.
 
+choice BT_HAS_HEARING_AID_TYPE_CHOICE
+	prompt "Hearing Aid Type selection"
+	help
+	  Select the Hearing Aid Type to compile.
+
+config BT_HAS_HEARING_AID_MONAURAL
+	bool "Monaural Hearing Aid"
+
+config BT_HAS_HEARING_AID_BINAURAL
+	depends on BT_CSIP_SET_MEMBER
+	bool "Binaural Hearing Aid"
+
+config BT_HAS_HEARING_AID_BANDED
+	bool "Banded Hearing Aid"
+
+endchoice # BT_HAS_HEARING_AID_TYPE_CHOICE
+
+choice BT_HAS_HEARING_AID_LOCATION
+	prompt "Hearing Aid Device Location"
+	help
+	  Select the Hearing Aid Device location.
+
+config BT_HAS_HEARING_AID_LEFT
+	bool "Left Ear"
+
+config BT_HAS_HEARING_AID_RIGHT
+	bool "Right Ear"
+
+endchoice # BT_HAS_HEARING_AID_LOCATION
```

Comment:
Since this is in a sample, I suggest we remove the `BT_HAS` and `BT` prefix, as they could otherwise clash with BT subsystem Kconfigs.

Reply:
Makes sense - I will change the prefix to the HAP_HA_* used above